fml/file.h

```diff
@@ -73,12 +78,37 @@ bool WriteAtomically(const fml::UniqueFD& base_directory,
                      const char* file_name,
                      const Mapping& mapping);
 
+using FileVisitor = std::function<void(const fml::UniqueFD& directory,
+                                       const std::string& filename)>;
+
+/// Call `visitor` on all files inside the `directory` non-recursively. The
+/// trivial file "." and ".." will not be visited.
+///
+/// If recursive visiting is needed, call `VisitFiles` inside the `visitor`, or
+/// use our helper method `VisitFilesRecursively`.
+///
+/// @see `VisitFilesRecursively`.
+void VisitFiles(const fml::UniqueFD& directory, const FileVisitor& visitor);
```

Comment:
You don't need to pass closures by const reference. Just `FileVisitor visitor` is fine.

Reply:
Done.